src/main/Validator.js

```diff
@@ -80,6 +80,7 @@ const configDataSchemaV2 = Joi.object({
   autostart: Joi.boolean().default(true),
   spellCheckerLocale: Joi.string().regex(/^[a-z]{2}-[A-Z]{2}$/).default('en-US'),
   darkMode: Joi.boolean().default(false),
+  downloadLocation: Joi.string().regex(/(\\\\?([^\\/]*[\\/])*)([^\\/]*)$/).default(`/Users/${process.env.USER}/Downloads`),
```

Comment:
@jupenur do you think we need to specify some regex for security reasons? if not, I'd prefer to remove the regex as there are many different paths escenarios and we might not cover some.

Reply:
No, I don't think there's any need.